br/pkg/lightning/backend/kv/base.go

```diff
@@ -94,6 +95,14 @@ func (row RowArrayMarshaller) MarshalLogArray(encoder zapcore.ArrayEncoder) erro
 				return err
 			}
 		}
+		if len(str) > 512*1024 {
```

Comment:
make is a const? its used in several place

Reply:
modified